File: backend/waitingNow/src/main/java/com/waiting/waitingnow/domain/WaitingVO.java
package com.waiting.waitingnow.domain;

import java.util.Date;

public class WaitingVO {
    private int waitingNumber;
    private int waitingCustomerNumber;
    private String waitingPhone;
    private int waitingPeople;
    private String waitingDate;
    private String waitingAvailable;
    private int memberNumber;

    public int getWaitingNumber() {
        return waitingNumber;
    }

    public void setWaitingNumber(int waitingNumber) {
        this.waitingNumber = waitingNumber;
    }

    public int getWaitingCustomerNumber() {
        return waitingCustomerNumber;
    }

    public void setWaitingCustomerNumber(int waitingCustomerNumber) {
        this.waitingCustomerNumber = waitingCustomerNumber;
    }

    public String getWaitingPhone() {
        return waitingPhone;
    }

    public void setWaitingPhone(String waitingPhone) {
        this.waitingPhone = waitingPhone;
    }

    public int getWaitingPeople() {
        return waitingPeople;
    }

    public void setWaitingPeople(int waitingPeople) {
        this.waitingPeople = waitingPeople;
    }

    public String getWaitingDate() {
        return waitingDate;
    }

    public void setWaitingDate(String waitingDate) {
        this.waitingDate = waitingDate;
    }

    public String getWaitingAvailable() {
        return waitingAvailable;
    }

    public void setWaitingAvailable(String waitingAvailable) {
        this.waitingAvailable = waitingAvailable;
    }

    public int getMemberNumber() {
        return memberNumber;
    }

    public void setMemberNumber(int memberNumber) {
        this.memberNumber = memberNumber;
    }
}

File: flutter/waitingnow/lib/src/Domain/WaitingVO.dart
class WaitingVO {
  int? waitingNumber;
  int? waitingCustomerNumber;
  String? waitingPhone;
  int? waitingPeople;
  String? waitingDate;
  int? waitingAvailable;
  int? memberNumber;

  WaitingVO(){}

  Map<String, dynamic> toJson() {
    return {
      'waitingNumber': waitingNumber,
      'waitingCustomerNumber': waitingCustomerNumber,
      'waitingPhone': waitingPhone,
      'waitingPeople': waitingPeople,
      'waitingDate': waitingDate,
      'waitingAvailable': waitingAvailable,
      'memberNumber': memberNumber,
    };
  }

  WaitingVO.fromJson(Map m) {
    waitingNumber = m['waitingNumber'];
    waitingCustomerNumber = m['waitingCustomerNumber'];
    waitingPhone = m['waitingPhone'];
    waitingPeople = m['waitingPeople'];
    waitingDate = m['waitingDate'];
    waitingAvailable = m['waitingAvailable'];
    memberNumber = m['memberNumber'];
  }

  printWaitings(){
    print("waiingNumber : " + waitingNumber.toString() + " / waitingCustomerNumber : " + waitingCustomerNumber.toString() + " / waitingPhone : " + waitingPhone.toString() + " / waitingAvailable : " + waitingAvailable.toString());
  }
}